google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -165,13 +165,16 @@ def visit_FLOAT(self, type_, **kw):
         return "FLOAT64"
 
     def visit_TEXT(self, type_, **kw):
-        return "STRING({})".format(type_.length)
+        return "STRING({})".format(type_.length or "MAX")
 
     def visit_ARRAY(self, type_, **kw):
         return "ARRAY<{}>".format(self.process(type_.item_type, **kw))
 
     def visit_BINARY(self, type_, **kw):
-        return "BYTES"
+        return "BYTES({})".format(type_.length or "MAX")
+
+    def visit_large_binary(self, type_, **kw):
+        return "BYTES({})".format(type_.length or "MAX")
```

Comment:
The last SQLAlchemy version says that `Binary` type is deprecated, and `LargeBinary` should be used instead. The `LargeBinary` type will pass `BLOB` into SQL, so I'm overriding the method to make it `BYTES`.